src/Main.java

```diff
@@ -1,5 +1,67 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        byte a = 127;
+        short b = -32768;
+        int c = 2147483647;
+        long d = -9223372036854775808L;
+        float e = 1.1234567f;
+        double f = 1.1234567891234567;
+        System.out.println("Значение переменной «a» с типом byte равно " + a);
+        System.out.println("Значение переменной «b» с типом short равно " + b);
+        System.out.println("Значение переменной «c» с типом int равно " + c);
+        System.out.println("Значение переменной «d» с типом long равно " + d);
+        System.out.println("Значение переменной «e» с типом float равно " + e);
+        System.out.println("Значение переменной «f» с типом double равно " + f);
+
+        float q = 27.12f;
+        long w = 987678965549L;
+        float o = 2.786f;
+        short r = 569;
+        short t = -159;
+        short y = 27897;
+        byte u = 67;
+        System.out.println(q);
+        System.out.println(w);
+        System.out.println(o);
+        System.out.println(r);
+        System.out.println(t);
+        System.out.println(y);
+        System.out.println(u);
+
+        byte studentsOfTheFirstTeacher = 23;
+        byte studentsOfTheSecondTeacher = 27;
+        byte studentsOfTheThirdTeacher = 30;
+        short totalSheets = 480;
+        System.out.println("На каждого ученика рассчитано " + totalSheets / (studentsOfTheFirstTeacher + studentsOfTheSecondTeacher + studentsOfTheThirdTeacher) + " листов бумаги");
+
+        byte equipmentPerformancePerMinute = 8;
+        System.out.println("За 20 минут машина произвела " + equipmentPerformancePerMinute * 20 + " штук бутылок");
+        System.out.println("За сутки машина произвела " + equipmentPerformancePerMinute * 1440 + " штук бутылок");
+        System.out.println("За три дня машина произвела " + equipmentPerformancePerMinute * 4320 + " штук бутылок");
+        System.out.println("За один месяц машина произвела " + equipmentPerformancePerMinute * 44640 + " штук бутылок");
+
+        byte totalCansOfPaint = 120;
+        byte whitePaint = 2;
+        byte brownPaint = (byte) (whitePaint * 2);
```

Comment:
зачем здесь вычисления? число 4 было дано, значит сразу его и надо записать в переменную
```suggestion
        byte brownPaint = 4;
```